### PR TITLE
Assignment6

### PR DESCRIPTION
**Pull request by 7hawks**

### CODE REVIEW

**Comment by CDobbins1** [comment]
Reviewing this.

File: SecretSanta/test/SecretSanta.Api.Tests/Controllers/BaseApiControllerTests.cs [hunk 1]
```diff
@@ -48,7 +48,7 @@ public async Task Get_WhenEntityDoesNotExist_ReturnsNotFound()
             TService service = new TService();
             BaseApiController<TDto, TInputDto> controller = CreateController(service);
 
-            IActionResult result = await controller.Get(1);
+            var result = await controller.Get(1);
 
             Assert.IsTrue(result is NotFoundResult);
```
Comment (CDobbins1) [inline SecretSanta/test/SecretSanta.Api.Tests/Controllers/BaseApiControllerTests.cs]:
```suggestion
            Assert.IsTrue(result.Result is NotFoundResult);
```
This should fix the failing tests.

File: SecretSanta/test/SecretSanta.Api.Tests/Controllers/BaseApiControllerTests.cs [hunk 2]
```diff
@@ -61,7 +61,7 @@ public async Task Get_WhenEntityExists_ReturnsItem()
             service.Items.Add(entity);
             BaseApiController<TDto, TInputDto> controller = CreateController(service);
 
-            IActionResult result = await controller.Get(entity.Id);
+            var result = await controller.Get(entity.Id);
 
             var okResult = result as OkObjectResult;
```
Comment (CDobbins1) [inline SecretSanta/test/SecretSanta.Api.Tests/Controllers/BaseApiControllerTests.cs]:
```suggestion
            var okResult = result.Result as OkObjectResult;
```
This should fix the failing tests.

File: SecretSanta/src/SecretSanta.Web/Views/Shared/_Layout.cshtml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        }
+    </style>
+</head>
+<body>
+    <header>
+        <nav>
+            <ul>
+                <li><a asp-controller="Users" asp-action="index">Users</a></li>
+                <li><a asp-controller="Gifts" asp-action="index">Gifts</a></li>
+                <li><a asp-controller="Groups" asp-action="index">Groups</a></li>
```
Comment (CDobbins1) [inline SecretSanta/src/SecretSanta.Web/Views/Shared/_Layout.cshtml]: Missing navigation link to Home.

File: SecretSanta/src/SecretSanta.Web/Views/Shared/_Layout_Template.cshtml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        input.input-validation-error {
+            border-color: red;
+        }
+    </style>
+</head>
+<body>
+    <header>
+        <nav>
+            <ul>
+                <li><a asp-controller="Users" asp-action="index">Users</a></li>
```
Comment (breaman) [inline SecretSanta/src/SecretSanta.Web/Views/Shared/_Layout_Template.cshtml]: Looks like your _Layout_Template doesn't match your _Layout file. Remember, the _Layout file will get deleted each time you run the webpack scripts.

**Pull request closed without merging**